### ISSUE TITLE
How to transfer data to custom-actions components?

### ISSUE THREAD

**Issue by a1lan1** [ratiw/vuetable-2#534]
Hey. I can not figure out how to transfer data from the main component to the custom component actions?

// MainComponent.vue
```
<template>
  <div>
    <filter-bar />
    <vuetable
      ref="vuetable"
      :actions="actions"
      :api-url="..."
      :fields="fields"
...
    />
```

// js
```
<script>
...
import CustomActions from './table/CustomActions'
import FilterBar from './table/FilterBar'

window.Vue.component('custom-actions', CustomActions)
...
```

Thanks!

**Comment by ratiw** [comment]
There is no `actions` property in Vuetable. Did you go through the tutorial yet?

**Comment by a1lan1** [comment]
Yes, I went through the tutorial and could not figure out how to transfer the value of a variable from the Vuetable component to the actions component

**Comment by a1lan1** [comment]
I do not want to show in all tables, all action buttons. In some tables I can only show one button, in other tables, other buttons. I would like the input parameters to determine which buttons should be shown in this table.

**Comment by ratiw** [comment]
Can you setup a sample project in CodeSandbox to demonstrate your problem so I can have a look at it? Based on what you've shown, I guess there is something incorrect in the field definition but I can't say for sure.
